L1Trigger/TrackFindingTracklet/src/PurgeDuplicate.cc

```diff
@@ -286,24 +278,24 @@ void PurgeDuplicate::execute(std::vector<Track>& outputtracks, unsigned int iSec
 
             // Fill duplicate map
             if (nShareLay >= settings_.minIndStubs()) {  // For number of shared stub merge condition
-              dupMap[itrk][jtrk] = true;
-              dupMap[jtrk][itrk] = true;
+              dupMap.at(itrk).at(jtrk) = true;
+              dupMap.at(jtrk).at(itrk) = true;
             }
           }
         }
 
         // Check to see if the track is a duplicate
         for (unsigned int itrk = 0; itrk < numStublists; itrk++) {
```

Comment:
why this loop at all?
just set noMerge[itrk] = true;  where one set the dupMap.